apps/sms/views/conversation/js/conversation.js

```diff
@@ -707,6 +696,15 @@ var ConversationView = {
     if (Navigation.isCurrentPanel('thread-list')) {
       // We don't want to clean these things when moving from composer to
       // conversation
+      // Revoke thumbnail URL for every image attachment rendered within thread
+      // Executed only when moving out of a conversation.
+      var nodes = this.container.querySelectorAll(
```

Comment:
good point

I think I'll extract `this.container.textContent = '';` to a separate function, replace it everywhere we use it (2 locations, for now), and add this code there